__tests__/app.test.js

```diff
@@ -24,9 +25,35 @@ describe("GET/api/topics", () => {
                         description: expect.any(String)
                     }));
                 })
-            } else {
-                expect(body.topics).toEqual([]);
             }
         });
     });
-});
\ No newline at end of file
+});
+
+describe("GET /api", () => {
+    it("should respond with status code 200 and an object describing all the available endpoints", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({body}) => {
+             expect(body).toEqual(endpoints);
+        });
+    });
+    it("should check that the response object is of the correct format", () => {
+        return request(app)
+        .get("/api/")
+        .then(({body}) => {
+            if (Object.keys(body).length > 0) {
+                for (let api in body) {
```

Comment:
I like the loop here to check each object has the expected keys however since your test above is checking the body equals the endpoints file thats been imported this second test isn't required.